packages/react/src/component-tests/IcFooter/IcFooter.cy.tsx

```diff
@@ -0,0 +1,458 @@
+            <IcFooterLink href="/">Styles</IcFooterLink>
+            <IcFooterLink href="/">Components</IcFooterLink>
+            <IcFooterLink href="/">Patterns</IcFooterLink>
+          </IcFooterLinkGroup>
+          <IcFooterLinkGroup slot="link" groupTitle="Grouped heading">
+            <IcFooterLink href="/">Accessibility</IcFooterLink>
+            <IcFooterLink href="/">Styles</IcFooterLink>
+            <IcFooterLink href="/">Components</IcFooterLink>
+            <IcFooterLink href="/">Patterns</IcFooterLink>
+          </IcFooterLinkGroup>
+          <IcFooterLinkGroup slot="link" groupTitle="Grouped heading">
+            <IcFooterLink href="/">Accessibility</IcFooterLink>
+            <IcFooterLink href="/">Styles</IcFooterLink>
+            <IcFooterLink href="/">Components</IcFooterLink>
+            <IcFooterLink href="/">Patterns</IcFooterLink>
+          </IcFooterLinkGroup>
+        </IcFooter>
+      );
+      cy.viewport(1440, 750);
+      cy.compareSnapshot("groupLinks", DEFAULT_TEST_THRESHOLD + 0.05);
+      cy.checkA11yWithWait();
+    });
+
+    it("render logo with links", () => {
+      mount(
+        <IcFooter description="The ICDS is maintained by the Design Practice Team. If you've got a question or want to feedback, please get in touch.">
+          {" "}
+          <IcFooterLink slot="link" href="/">
+            Accessibility
+          </IcFooterLink>
+          <IcFooterLink slot="link" href="/">
+            Styles
+          </IcFooterLink>
+          <IcFooterLink slot="link" href="/">
+            Components
+          </IcFooterLink>
+          <IcFooterLink slot="link" href="/">
+            Patterns
+          </IcFooterLink>
+          <div slot="logo" style={{ display: "flex" }}>
+            <IcFooterLink href="/">
+              <SlottedSVG
+                xmlns="http://www.w3.org/2000/svg"
+                height="48px"
+                viewBox="0 0 24 24"
+                width="48px"
+                fill="#FFFFFF"
+              >
+                <path
+                  d="M12 6.19L17 10.69V18.5H15V12.5H9V18.5H7V10.69L12 6.19ZM12 3.5L2 12.5H5V20.5H11V14.5H13V20.5H19V12.5H22L12 3.5Z"
+                  fill="currentColor"
+                />
+              </SlottedSVG>
+            </IcFooterLink>
+            <IcFooterLink href="/">
+              <SlottedSVG
+                xmlns="http://www.w3.org/2000/svg"
+                height="48px"
+                viewBox="0 0 24 24"
+                width="48px"
+                fill="#FFFFFF"
+              >
+                <path
+                  d="M12 6.19L17 10.69V18.5H15V12.5H9V18.5H7V10.69L12 6.19ZM12 3.5L2 12.5H5V20.5H11V14.5H13V20.5H19V12.5H22L12 3.5Z"
+                  fill="currentColor"
+                />
+              </SlottedSVG>
+            </IcFooterLink>
+          </div>
+        </IcFooter>
+      );
+      cy.compareSnapshot("logoWithLinks", DEFAULT_TEST_THRESHOLD + 0.5);
```

Comment:
I think the threshold for this is missing a 0, it seems to pass every time no matter what changes
```suggestion
      cy.compareSnapshot("logoWithLinks", DEFAULT_TEST_THRESHOLD + 0.05);
```